### PR TITLE
feat - added different types of buttons

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added different types of buttons 

Live Link : https://easy-ui-prathmesh.netlify.app/componets/buttons/buttons.html#floatingButton

### CODE REVIEW

File: componets/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.btn {
+  margin: 8px;
+  padding: 15px;
```
Comment (Mohasin-Haque) [inline componets/buttons/buttons.css:3]: If possible please use relative units.

File: componets/buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+.btn-primary:hover {
+  background-color: var(--secondary-color);
+}
+.btn-success {
+  background-color: var(--success-green-color);
+  color: var(--background-color);
+}
+.btn-success:hover {
+  background-color: #34d399;
```
Comment (Mohasin-Haque) [inline componets/buttons/buttons.css]: can be done using variables.

File: componets/buttons/buttons.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+}
+.btn-warning {
+  background-color: var(--warning-red-color);
+  color: var(--background-color);
+}
+.btn-warning:hover {
+  background-color: #f87171;
```
Comment (Mohasin-Haque) [inline componets/buttons/buttons.css]: make it variable and use it.

**Comment by tanishk15sharma** [comment]
good work!

File: componets/buttons/buttons.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Mohasin-Haque) [inline componets/buttons/buttons.html:6]: can add more meta information to give it a personalise touch.. 
like <meta name="author" content="Your Name">
<meta name="description" content="about your ui">

**Review by MarkVed17 — APPROVED** [review]
Good work! Just that the sticky header takes off some body content when viewed on mobile. Adjust it & you’re good to go.

File: componets/buttons/buttons.css (hunk 2 above)
Comment (shilpe26) [inline componets/buttons/buttons.css]: use variable naming and then use it

**Review by Mohasin-Haque — COMMENTED** [review]
Please work on the commented review. Try to make it sanity.

File: componets/buttons/buttons.html [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+            <button class="btn">Default</button>
+            <button class="btn btn-primary">Primary</button>
+            <button class="btn btn-warning">Warning</button>
+            <button class="btn btn-success">Success</button>
+            <button class="btn btn-dark">Dark</button>
+          </div>
+          <div class="component_code">
+            <iframe
+              src="https://carbon.now.sh/embed/UkIiypETiD7eUYDafHP2"
+              style="
```
Comment (shilpe26) [inline componets/buttons/buttons.html]: don't apply styling in Html page use CSS file for it

**Review by shilpe26 — COMMENTED** [review]
apply beautify to your code so that it becomes easy to read

File: componets/buttons/buttons.html (hunk 5 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.html]: when we take iframe URL from carbon height  is different  for each iframe so how  can I achieve  this

**Comment by prathmeshjagtap** [comment]
> Good work! Just that the sticky header takes off some body content when viewed on mobile. Adjust it & you’re good to go.

yes it is not responsive yet I was thinking  of doings the components  first once dowe with components  I will do that

File: componets/buttons/buttons.css (hunk 2 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.css]: yes did this change thanks for the suggestion

File: componets/buttons/buttons.css (hunk 1 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.css:3]: padding and margin is less than 1 rem so I thought  it is better to keep in fixed and I don't  want margin to increase  with rem units , is it fine

File: componets/buttons/buttons.css (hunk 2 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.css]: yes did this change thanks for the suggestion

File: componets/buttons/buttons.css (hunk 3 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.css]: yes did this change thanks for the suggestion

**Comment by prathmeshjagtap** [comment]
> apply beautify to your code so that it becomes easy to read

yes I have added it

File: componets/buttons/buttons.html (hunk 4 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.html:6]: I don't know more about it i will read about it and do the changes

File: componets/buttons/buttons.html (hunk 5 above)
Comment (shilpe26) [reply componets/buttons/buttons.html]: you can add class to your frame and then give style.
Rest is all right.

File: componets/buttons/buttons.html (hunk 4 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.html:6]: ok i will do it thanks

File: componets/buttons/buttons.html (hunk 5 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.html]: ok will do it thanks

**Review by NtshVrm — APPROVED** [review]

File: componets/buttons/buttons.html (hunk 5 above)
Comment (prathmeshjagtap) [reply componets/buttons/buttons.html]: done

**Review by shilpe26 — APPROVED** [review]
Now its better you can proceed further.👍🏻

**Pull request merged**